src/storage/src/source/reclock.rs

```diff
@@ -942,11 +662,15 @@ mod tests {
             status_shard: None,
         };
 
-        let (operator, initial_batch) = ReclockOperator::new(
-            Arc::clone(persist_cache),
+        let clock_stream = futures::stream::iter((0..).map(|seconds| {
+            let ts = Timestamp::from(seconds * 1000);
+            let upper = Antichain::from_elem(ts.step_forward());
+            (ts, upper)
+        }));
+
+        let remap_handle = crate::source::reclock::compat::PersistHandle::new(
```

Comment:
we test with compat...can you add a `TODO` to test with a non-persist impl?

Reply:
will do. I can make a loopback `RemapHandle` and run the tests with that